CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Maintenance
 ### Refactoring
 
-## [Unreleased 2.x](https://github.com/opensearch-project/neural-search/compare/2.7...2.x)
+## [Unreleased 2.x](https://github.com/opensearch-project/neural-search/compare/2.10...2.x)
 ### Features
 * Added Score Normalization and Combination feature ([#241](https://github.com/opensearch-project/neural-search/pull/241/))
```

Comment:
When you add item in release note, it needs to be removed from CHANGELOG.

Reply:
I want to remove it from main once the code freeze is done.

Reply:
Any reason of doing it? Removing them later increase a chance of mistake. You need to inspect each item if they are added in release note or not when new PR comes in before you remove them.

Reply:
I am not 100% sure about this thing here. I was thinking the CHANGELOG.md needs to be update when we upgrade the version from 2.10 to 2.11 in this case. Give that 2.11 is not ready, I cannot update version to lets say 2.11.